### DISCUSSION TITLE
Bulk import local videos and embed in html video format within tiddlers.

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#127] [Ideas]
I want to import multiple local video files, mostly mp4 videos (say 4 or 5 videos) in to a single tiddlers in a single clipping session using tiddlyclip. I have a working rule for importing single video file at one time in html format - Check these two tiddlers - [1](https://all-demo-wiki.tiddlyhost.com/#LocalDefaultmp4Rule) and [2](https://all-demo-wiki.tiddlyhost.com/#LocalDefaultmp4FieldModifiers).

![image](https://github.com/buggyj/tiddlyclip-plugin/assets/67494083/367fe190-0f2f-469e-8f1c-eeee4c02be97)

Can a rule be made to import multiple files at once. Something similar to this https://github.com/buggyj/tiddlyclip-plugin/discussions/120

**Comment by buggyj** [comment]
multiple import of vids - use the same as in #120 

[createvids.json.zip](https://github.com/buggyj/tiddlyclip-plugin/files/12884262/createvids.json.zip)

**Reply by arunnbabu81** [reply]
@buggyj Thank you . it works perfectly
